components/datatypes/contentaudit.schema.json

```diff
@@ -0,0 +1,61 @@
+{
+  "meta:license": [
+    "Copyright 2025 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/contentaudit",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Content Audit",
+  "type": "object",
+  "meta:abstract": true,
+  "description": "Content audit.",
```

Comment:
Could you provide a more meaningful description? (that will help in the field discovery)